apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -427,7 +427,13 @@
           "approved": "KYC Approved!",
           "closed": "KYC Rejected"
         }
-      }
+      },
+      "filter": {
+        "all": "All",
```

Comment:
Is this viewable as part of this PR or ignore until the DEX/filter PR?

Reply:
yes, I am applying here the changes too advance the translations